lib/mix/tasks/git_ops.release.ex

```diff
@@ -275,7 +279,26 @@ defmodule Mix.Tasks.GitOps.Release do
     end
   end
 
-  defp commits_with_type(config_types, commits, text, log?) do
+  defp commits_with_allowed_tags(commits, :any), do: commits
+
+  defp commits_with_allowed_tags(commits, allowed_tags) do
+    case Enum.find(commits, fn %{type: type} -> type == "TAGS" end) do
+      nil ->
+        # If the commits don't have a tag, we allow it to go through by default
+        commits
```

Comment:
we should add a config, i.e `allow_untagged?: true/false`

Reply:
Any suggestion on where that should be? is it OK if it is in the "root" level of the :git_ops config?

`config :git_ops, allows_untagged?: false`